package.json

```diff
@@ -65,6 +70,7 @@
     "broccoli-merge-trees": "^1.1.0",
     "broccoli-string-replace": "^0.1.1",
     "chalk": "^1.1.0",
+    "electron-forge": "^2.8.0",
     "electron-packager": "^8.3.0",
```

Comment:
Good catch. There are several others that can be cleaned out too, like winstaller / ect